tests/test_mesh.py

```diff
@@ -787,12 +787,23 @@ def test_incidence(mesh):
 
     p2t = mesh.p2t
     for itr in range(0, 50, 3):
-       assert np.sum((mesh.t == itr).any(axis=0)) == len(p2t[:, itr].data)
+        assert np.sum((mesh.t == itr).any(axis=0)) == len(p2t[:, itr].data)
 
     p2f = mesh.p2f
     for itr in range(0, 50, 3):
-       assert np.sum((mesh.facets == itr).any(axis=0)) == len(p2f[:, itr].data)
+        assert np.sum((mesh.facets == itr).any(axis=0)) == len(p2f[:, itr].data)
 
+    p2e = mesh.p2e
```

Comment:
These additional tests should run only for `MeshTet` and `MeshHex`.

Reply:
Added a check for those mesh objects in a new commit